src/Saunter/AsyncApiOptions.cs

```diff
@@ -49,16 +50,22 @@ public class AsyncApiOptions
         /// <summary>
         /// Settings related to the JSON Schema generation.
         /// </summary>
-        public JsonSchemaGeneratorSettings JsonSchemaGeneratorSettings { get; set; } = new JsonSchemaGeneratorSettings()
+        public AsyncApiSchemaOptions SchemaOptions { get; set; } = new AsyncApiSchemaOptions();
+    }
+
+    public class AsyncApiSchemaOptions : JsonSchemaGeneratorSettings
```

Comment:
Using an own AsyncApiSchemaOptions class which inherits from JsonSchemaGeneratorSettings so that we can enforce a default SchemaType in the ctor... (not sure what it should be yet however)

(My proposal would be to rename AsyncApiSchemaOptions to AsyncApiSchemaGeneratorSettings
and AsyncApiOptions to AsyncApiGeneratorSettings - but that's up to you, would just be more in sync with NJsonSchema and NSwag.)

Reply:
I agree it makes sense to rename. 
This should be fine to make that change as these classes are exclusively used via `Action<AsyncApiOptions> setupAction`, so changing the class name should not impact any users.